W-18291493 install cursor and test alt text updates (#599)
mulesoft/docs-api-manager: modules/ROOT/pages/api-gateway-domain.adoc

You can check that the domain is associated to your project by
viewing the mule-project.xml file of your project.

image::api-gateway-mule-project.png["Mule project configuration showing API Gateway domain association settings"]

> A Gateway 2.x domain project is identical to the domain that exists in Cloudhub and in your API Gateway On Premises by default. It's necessary for being able to deploy your app to the Anypoint Studio server under the same conditions as when deployed to production.

Reviewer: ```suggestion
By default, a Gateway 2.x domain project is identical to the domain that exists in CloudHub and in your on-premises API Gateway. You must deploy your app to the Anypoint Studio server using the same configuration you deployed the app to production with.  
```

Author: Yeah, I wasn't touching the content.
